### ISSUE TITLE
Add potion throwing skill

### ISSUE THREAD

**Issue by zerafox** [MyPetORG/MyPet#174]
Not sure if this is possible, but I had a wonderful idea to add to my pet trees as I was working on them.

Make it so pets could use potions on their owners?

I'd love to be able to create a healing tree for the pets on my server. If a pet could use potions/splash potion on/for their owners that would open up a whole new window to the usage of the pets I think.

From what I can tell, you're using abilities and features that Minecraft gives us already as far as skills for the pets and I think a flat out "heal spell" would be too much to ask for, if it were even possible to do. XD

**Comment by zerafox** [comment]
Just curious, is there any news on this being added to MyPet in the near future? :P

**Comment by xXKeyleXx** [comment]
Not really :/ I don't have the motivation to work on the plugin and other project have a higher priority right now.

**Comment by zerafox** [comment]
Fair enough. ^.^ Certainly no rush, and was just touching base on the subject.
As always, thank you for your wonderful work. <3

**Comment by zerafox** [comment]
*gently pokes*
Any more news on how this is going?

**Comment by xXKeyleXx** [comment]
I know I said that I will add it someday but I don't know if it will ever happen. Sorry :/

**Comment by zerafox** [comment]
Still hopeful this will make it in some day.. :(
Haha

**Comment by UserDerezzed** [comment]
Added Potion skill in MyPet 4.0.0-alpha-27
